spec/unit/outputs/kafka_spec.rb

```diff
@@ -102,14 +105,14 @@
     before do
       count = 0
       expect_any_instance_of(org.apache.kafka.clients.producer.KafkaProducer).to receive(:send)
-        .exactly(1).times
-        .and_wrap_original do |m, *args|
+        .exactly(1).times.and_wrap_original do |m, *args|
```

Comment:
Similarly, we don't use `m` here so we don't need to wrap original.

```suggestion
        .exactly(1).times do |*args|
```